pkgs/by-name/mc/mcpelauncher-client/dont_download_glfw_client.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/ext/glfw.cmake b/ext/glfw.cmake
+index 5487f32..feeb89f 100644
+--- a/ext/glfw.cmake
++++ b/ext/glfw.cmake
+@@ -1,18 +1,2 @@
+-include(FetchContent)
+-
+-set(GLFW_BUILD_DOCS OFF CACHE BOOL "" FORCE)
+-set(GLFW_BUILD_TESTS OFF CACHE BOOL "" FORCE)
+-set(GLFW_BUILD_EXAMPLES OFF CACHE BOOL "" FORCE)
+-set(GLFW_INSTALL OFF CACHE BOOL "" FORCE)
+-
+-FetchContent_Declare(
```

Comment:
This patch should also not be necessary as it's also using FetchContent (you can probably get away with setting the source dir to `glfw.src`)

Reply:
This is intentional as it avoids building glfw.